layouts/_default/single.html

```diff
@@ -17,6 +17,15 @@ <h1 class="font-bold text-3xl text-primary-text">{{ .Title }}</h1>
         <div class="content">
             {{ .Content}}
         </div>
+
+        {{ if isset $.Params "carousel" }}
+        {{ if $.Params.carousel.enable }}
+        {{ if gt (len $.Params.carousel.images) 0 }}
+        {{ partial "components/carousel" . }}
+        {{ end }}
+        {{ end }}
+        {{ end }}
```

Comment:
As comment in `carousel.md`, a single `{{ if .Params.carousel }}` or `{{ with .Params.carousel }}` is great.

Reply:
Done in fabf93a